brewtils/models.py

```diff
@@ -782,34 +782,35 @@ class Event(BaseModel):
     def __init__(
         self,
         name=None,
-        payload=None,
-        error=None,
+        namespace=None,
+        garden=None,
         metadata=None,
         timestamp=None,
-        namespace=None,
+        payload_type=None,
+        payload=None,
+        error=None,
+        error_message=None,
     ):
         self.name = name
-        self.payload = payload
-        self.error = error
+        self.namespace = namespace
+        self.garden = garden
```

Comment:
If you have the name space, why do you need the garden information for events? You should be able to do this mapping in the framework for routing.

Reply:
Address my comment in the Beer Garden PR first, then we can decide if this comment was required.